### PR TITLE
docs: @aws-amplify/storage Access all files from storage using list API

### PR DESCRIPTION
**Pull request by kvramyasri7**
_Issue #, if available:_

_Description of changes:_

Added instruction on how to access all files from S3 using list API.

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: src/fragments/lib/storage/js/list.mdx [hunk 1]
```diff
@@ -95,3 +95,19 @@ Storage.list('photos/', { level: 'private' })
     .then(result => console.log(result))
     .catch(err => console.log(err));
 ```
+
+## Access ALL files 
+
+To list ALL files from S3, you can set maxKeys to 'ALL'.
```
Comment (abdallahshaban557) [inline src/fragments/lib/storage/js/list.mdx:101]: nit: To get a list of all files in your S3 bucket under a specific prefix, you can set `maxKeys` to `ALL`

**Review by abdallahshaban557 — COMMENTED** [review]
Nit - otherwise looks good!

**Review by abdallahshaban557 — APPROVED** [review]
LGTM! Thank you!

**Review by ashika01 — APPROVED** [review]
LGTM

File: src/fragments/lib/storage/js/list.mdx [hunk 2]
```diff
@@ -95,3 +95,19 @@ Storage.list('photos/', { level: 'private' })
     .then(result => console.log(result))
     .catch(err => console.log(err));
 ```
+
+## Access all files 
+
+To get a list of all files in your S3 bucket under a specific prefix, you can set maxKeys to ALL.
```
Comment (elorzafe) [inline src/fragments/lib/storage/js/list.mdx:101]: I would mention what is the default, what do you think?

Comment (kvramyasri7) [reply src/fragments/lib/storage/js/list.mdx:101]: Yeah. that would totally makes sense.

**Review by arundna — APPROVED** [review]

**Pull request merged**